package/pyproject.toml

```diff
@@ -125,5 +125,10 @@ MDAnalysis = [
 [tool.black]
 line-length = 79
 target-version = ['py310', 'py311', 'py312']
-extend-exclude = '.'
+include = '''
+(
+tables\.py
+| due\.py
+)
+'''
```

Comment:
Let's start simple, with only two files. Other files or matching regexp can be added with the `|` operator.